### DISCUSSION TITLE
Can I upgrade non-Docker from 4.0.2 to 4.1.1 without first upgrading to 4.0.3 or 4.1.0?

### DISCUSSION THREAD

**Discussion by mtbluedog** [mastodon/mastodon#24335] [Q&A]
I've read the release notes and there are no dependency changes from 4.0.2 to 4.1.1.
So in my 4.0.3 site, can I upgrade to 4.1.1 this way?:
`su - mastodon`
and
`cd /home/mastodon/live`
and then run
`git fetch && git checkout v4.1.1`
and then
`RAILS_ENV=production bundle exec rails assets:precompile`
and then
`systemctl restart mastodon-*`

**Comment by Lastorder-DC** [comment]
No, You'll need to run `bundle install && yarn install` because while dependency itself like postgresql or ruby is same, library version is bumped.(mostly security patches)
